.scripts/run_config.sh

```diff
@@ -22,21 +22,27 @@ do
   esac
 done
 
-root_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )/..
+root_path=$( cd "$(dirname "$(dirname "${BASH_SOURCE[0]}")")" && pwd -P )
 
 # Use jq to grab playbooks into a bash array in order of appearance, and database target into a var.
 playbooks=($(cat $CONFIG_PATH | jq -r '.data.playbooks[].playbook | @sh' | tr -d \'))
 target=$(cat $CONFIG_PATH | jq -r '.data.storage')
-model_path="$(dirname "${CONFIG_PATH}")/.."
+model_path=$( cd "$(dirname "$(dirname "${CONFIG_PATH}")")" && pwd -P )
 
 set -e
 
+if [ -d "${root_path}/tmp" ]; then
```

Comment:
Why require that the `tmp` dir not exist, and destroy everything in it, rather than just creating it if it doesn't exist?

Reply:
Strictly speaking there is no need. But assuming the dir structure of the repo, and considering `/tmp` to be temporarily there for credentials, i would prefer it to have separate concerns that e.g. output or any other pre-existing dir. Of course i could change that back, if we think there is no real reason for this distinction.

Reply:
I think there are two questions: 

1. Should the script fail if `tmp` exists already? 

I don't see a good reason. Even if we expect it to not exist, it's easy to make the workflow smoother by using `mkdir -p`, which won't fail if it exists.

2. Should we drop the entire `tmp` directory instead of just deleting one file?

I can see an argument that it's safe to drop the entire thing. Originally, I wanted it to also work if someone has manually added a JSON file containing their BQ credentials - in fact one needs to do this if using great_expectations or sql-runner directly, without our scripts for example.

However I'm not sure there's a strong argument for that now, as I changed a lot since then... Let's return to it.

Reply:
@colmsnowplow i'm sorry, indeed i had misunderstood the issue. In the "Address review changes II" commit, i also tried to remove the use of `DB_PASSWORD` so as to have same env vars for both sql-runner and great-expectations.
Finally, i wasn't aware of the use case for storing in `bq_creds.json`, so, even though still a bit tempting, reverted the total removal of `tmp`. As far as i've tested the issue seems solved. Please, let me know if you also find so.

Reply:
Actually I think this one has been confusing because I realised recently that we have _two_ bugs:

1. The credentials weren't handled as expected
2. The `tmp` folder wasn't created when it didn't exist.

So it's my bad for first introducing the bugs, and second missing one of them and making it more confusing.

Reply:
I think we have made the changes here both to use `mkdir -p` and the env vars? If so we can resolve this.